frontend/src/scenes/max/QuestionSuggestions.tsx

```diff
@@ -8,11 +8,15 @@ import { sidePanelSettingsLogic } from '~/layout/navigation-3000/sidepanel/panel
 import { maxLogic } from './maxLogic'
 
 export function QuestionSuggestions(): JSX.Element {
-    const { visibleSuggestions, allSuggestionsLoading, dataProcessingAccepted } = useValues(maxLogic)
+    const { visibleSuggestions, allSuggestionsLoading, dataProcessingAccepted, tools } = useValues(maxLogic)
     const { askMax, shuffleVisibleSuggestions } = useActions(maxLogic)
     const { coreMemory, coreMemoryLoading } = useValues(maxSettingsLogic)
     const { openSettingsPanel } = useActions(sidePanelSettingsLogic)
 
+    if (tools.length > 0) {
+        return <></>
```

Comment:
What about having tool specific suggestions (hardcoded for now)?

Reply:
Yup, for example the `search_session_recordings` tool actually already has its hard-coded suggestions already registered. We just haven't hooked them up to `QuestionSuggestions` yet, but simple to do. Here the question is: What suggestions would be useful for SQL editor useful to get started, if any? @EDsCODE

Reply:
I think for SQL editor i'd leave empty for now. We would want to generate something relevant to the query in context but out of scope here